handlers.go

```diff
@@ -17,8 +18,46 @@ func newSpecialReportsHandler(service specialReportService) specialReportsHandle
 	return specialReportsHandler{service: service}
 }
 
+func (h *specialReportsHandler) count(writer http.ResponseWriter, req *http.Request) {
+	ids := h.service.getSpecialReportIds()
+	_, err := writer.Write([]byte(strconv.Itoa(len(ids))))
+	if err != nil {
+		log.Warnf("Couldn't write count to HTTP response. count=%d %v\n", len(ids), err)
+		writer.WriteHeader(http.StatusInternalServerError)
+	}
+}
+
+func (h *specialReportsHandler) ids(writer http.ResponseWriter, req *http.Request) {
+	ids := h.service.getSpecialReportIds()
+	writer.Header().Add("Content-Type", "text/plain")
+	if len(ids) == 0 {
+		writer.WriteHeader(http.StatusOK)
```

Comment:
Well I'd expect that an empty response for getting the ids is a failure, and I'd expect concept-publisher to report it as a failure.
But as this is high priority, you could refine it with a later PR